### ISSUE TITLE
Warnings fixen, wo möglich

### ISSUE THREAD

**Issue by marianoju** [maddosz12/pytorch_team_2#56]
@Andreas Aus dem TODO geht nicht hervor, um welche Warnings es geht, welche Dateien betroffen sind, und warum diese gefixt werden sollen.

**Comment by fenomeno711** [comment]
/anaconda3/envs/neue-fische/bin/python /Users/Andreas/PycharmProjects/pytorch_team_2/projects/imdb/imdb.py
/anaconda3/envs/neue-fische/lib/python3.6/site-packages/sklearn/ensemble/weight_boosting.py:29: DeprecationWarning: numpy.core.umath_tests is an internal NumPy module and should not be imported. It will be removed in a future NumPy release.
  from numpy.core.umath_tests import inner1d
/anaconda3/envs/neue-fische/lib/python3.6/site-packages/sklearn/preprocessing/label.py:95: DataConversionWarning: A column-vector y was passed when a 1d array was expected. Please change the shape of y to (n_samples, ), for example using ravel().
  y = column_or_1d(y, warn=True)
/anaconda3/envs/neue-fische/lib/python3.6/site-packages/sklearn/preprocessing/label.py:128: DataConversionWarning: A column-vector y was passed when a 1d array was expected. Please change the shape of y to (n_samples, ), for example using ravel().
  y = column_or_1d(y, warn=True)

**Comment by marianoju** [comment]
Die `DeprecationWarning` würde ich ignorieren. Ich sehe nicht, dass wir da etwas machen können: 
> `... numpy.core.umath_tests is an internal NumPy module...`
